modules/auxiliary/scanner/http/wp_depicter_sqli_cve_2025_2011.rb

```diff
@@ -0,0 +1,147 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Auxiliary::Scanner
+  include Msf::Exploit::Remote::HTTP::Wordpress
+  include Msf::Exploit::Remote::HTTP::Wordpress::SQLi
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'WordPress Depicter Plugin SQL Injection (CVE-2025-2011)',
+        'Description' => %q{
+          The Slider & Popup Builder by Depicter plugin for WordPress <= 3.6.1 is vulnerable to unauthenticated SQL injection via the 's' parameter in admin-ajax.php.
+        },
+        'Author' => [
+          'Muhamad Visat',     # Vulnerability Discovery
+          'Valentin Lobstein'  # Metasploit Module
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          ['CVE', '2025-2011'],
+          ['WPVDB', '6f894272-3eb6-4595-ae00-1c4b0c0b6564'],
+          ['URL', 'https://cloud.projectdiscovery.io/library/CVE-2025-2011'],
+          ['URL', 'https://plugins.trac.wordpress.org/browser/depicter/trunk/app/src/Controllers/Ajax/LeadsAjaxController.php?rev=3156664#L179']
+        ],
+        'Actions' => [['SQLi', { 'Description' => 'Perform SQL Injection via admin-ajax.php?s=' }]],
+        'DefaultAction' => 'SQLi',
+        'DefaultOptions' => { 'VERBOSE' => true, 'COUNT' => 1 },
+        'DisclosureDate' => '2025-05-08',
+        'Notes' => { 'Stability' => [CRASH_SAFE], 'SideEffects' => [IOC_IN_LOGS], 'Reliability' => [] }
```

Comment:
Could we run rubocop on this file?
Potentially:
`bundle exec rubocop -A modules/auxiliary/scanner/http/wp_depicter_sqli_cve_2025_2011.rb`